common/classify.py

```diff
@@ -61,14 +64,18 @@ def generate_training(self):
         self.X = X
         self.Y = Y
         self.data_size = len(X)
-        progress_logger.info("Finished concatenating training matrix.")
+        end_time = time.time()
+        min_elapsed, sec_elapsed = int((end_time - start_time) / 60), int((end_time - start_time) % 60)
+        progress_logger.info("Finished concatenating training matrix in {} minutes, {} seconds.".format(min_elapsed, sec_elapsed))
```

Comment:
Might be useful to count the number of iterations of the `os.walk()` loop (the number of dirs) and print a count here, so we can interpret the counts that occur later.